src/views/layouts/includes/f-item-header-content.html

```diff
@@ -0,0 +1,12 @@
+{{#if data.labels}}
+	<ul class="f-Item-labels f-u-listInline">
+		{{#each data.labels}}
+			<li><span class="f-Label f-Label--{{this}}">{{this}}</span></li>
```

Comment:
OK, I'm seeing the totoro example which answers that question, methinks. Default treatment is grey with displayed text...correct? That seems good if so.